src/rez/tests/test_utils.py

```diff
@@ -48,3 +51,20 @@ def test_unix_case_insensistive_platform(self):
         path = filesystem.canonical_path('/a/b/File.txt', platform)
         expects = '/a/b/file.txt'.replace('\\', os.sep)
         self.assertEqual(path, expects)
+
+class TestLoadModuleFromFile(TestBase):
```

Comment:
This should go in a `test_util.py`. `test_utils.py` is for things in `src/rez/utils`.